Adding `Display` for our `Writeable` types (#2518)
unicode-org/icu4x: utils/writeable/README.md

let message = WelcomeMessage { name: "Alice" };
assert_writeable_eq!(&message, "Hello, Alice!");

> // Types implementing `Writeable` are required to also implement `fmt::Display`.
> // This can be simply done by redirecting to the `Writeable` implementation:

Reviewer: ```suggestion
// Types implementing `Writeable` should often also implement `fmt::Display`.
// This can be simply done by redirecting to the `Writeable` implementation:
```
